index.d.ts

```diff
@@ -0,0 +1,173 @@
+declare module "bw-chk" {
```

Comment:
I think this isn't how you do things unless you're declaring types for a *different* module? (I don't fully recall, I like never write these things by hand for my own libraries). AFAIK this should either be a namespace (`declare namespace foo`) or this stuff should just be at the top-level

Reply:
Another lib with manual types as an example: https://github.com/pmndrs/postprocessing/blob/main/types/postprocessing.d.ts

Reply:
It works either way afaik. This is just declaring it as an ambient module rather than direct declarations for each export. I just copy pasted so it was easier for me. Whichever you prefer.

Another lib with manual types as an example: https://github.com/pmndrs/postprocessing/blob/main/types/postprocessing.d.ts

Reply:
Doesn't seem to be a common way of doing things (the 10 or so `@types` packages I looked at all don't do this, and the TS compiler doesn't generate typings in this form) so I'd prefer to just put these at the top level probably

Reply:
Finally on winter break 👍 Please let me know if the changes are suitable.